src/test/kotlin/org/rustSlowTests/cargo/runconfig/test/CargoTestNodeInfoTest.kt

```diff
@@ -52,26 +52,14 @@ class CargoTestNodeInfoTest : CargoTestRunnerTestBase() {
        assert_ne!(123, 123, "123 == 123");
     """, "123 == 123")
 
+    fun `test don't unescape error messages`() = checkErrors("""
+       assert_eq!("a\\\\b", "a\\b", "`a\\\\b` != `a\\b`");
+    """, "`a\\\\b` != `a\\b`")
+
     @MinRustcVersion("1.39.0")
-    fun `test successful output`() = checkOutput("""
```

Comment:
Let's keep this test and change expected output just to commit current behavior and not to write these tests again when we will fix problem with escaping